dot/telemetry/mailer_test.go

```diff
@@ -206,13 +177,14 @@ func TestInfiniteListener(t *testing.T) {
 func listen(w http.ResponseWriter, r *http.Request) {
 	c, err := upgrader.Upgrade(w, r, nil)
 	if err != nil {
-		log.Printf("Error %v\n", err)
+		fmt.Printf("Error %v\n", err)
 	}
+
 	defer c.Close()
 	for {
 		_, msg, err := c.ReadMessage()
 		if err != nil {
-			log.Printf("read err %v", err)
+			fmt.Printf("read err %v", err)
```

Comment:
same question here, this should fail the test and not log something (aka no-one reads test logs if it doesn't fail)